modules/configuring-ibm-cos.adoc

```diff
@@ -0,0 +1,138 @@
++
+[source,terminal]
+----
+$ ibmcloud plugin install cos -f
+----
+
+. Set a bucket name.
++
+[source,terminal]
+----
+$ BUCKET=<bucket_name>
+----
+
+. Set a bucket region.
++
+[source,terminal]
+----
+$ REGION=<bucket_region> <1>
+----
+<1> For example, `eu-gb`.
+
+. Create a resource group by running the following command:
++
+[source,terminal]
+----
+$ ibmcloud resource group-create <resource_group_name>
+----
+
+. Set the `target` resource group by running the following command:
++
+[source,terminal]
+----
+$ ibmcloud target -g <resource_group_name>
+----
+
+. Verify that the target resource group is correctly set, by running the following command. In the example output, the resource group is set to `Default`.
++
+[source,terminal]
+----
+$ ibmcloud target
+----
++
+.Example output
++
+[source,yaml]
+----
+API endpoint:     https://cloud.ibm.com
+Region:           
+User:             test-user
+Account:          Test Account (fb6......e95) <-> 2...122
+Resource group:   Default
+----
+
+. Set the resource group variable.
++
+[source,terminal]
+----
+$ RESOURCE_GROUP=<resource_group> <1>
+----
+<1> For example, `"default"`.
+
+. Create an {ibm-cloud-title} resource `service-instance` by running the following command. For more details, refer link:https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-provision[Choosing a plan and creating an instance].
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create \
+<service_instance_name> \// <1>
+<service_name> \// <2>
+<service_plan> \// <3>
+<region_name> // <4>
+----
+<1> Specify a name for the `service-instance`.
+<2> Specify the service name. Alternatively, you can also specify a service ID instead of the service name.
+<3> Specify the service plan for your {ibm-cloud-title} account.
+<4> Specify the region name. 
++
+.Example command
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create test-service-instance cloud-object-storage \ // <1>
+standard \
+global \
+-d premium-global-deployment // <2>
+----
+<1> `cloud-object-storage` is the service name.
+<2> The `-d` flag specifies the deployment name.
+
+. Extract the `service-instance-id` by running the following command:
++
+[source,terminal]
+----
+$ SERVICE_INSTANCE_ID=$(ibmcloud resource service-instance test-service-instance --output json | jq -r '.[0].id')
+----
+
+. Create a COS bucket by running the following command. The values for `$BUCKET`, `$SERVICE_INSTANCE_ID`, and `$REGION` will be replaced.
++
+[source,terminal]
+----
+$ ibmcloud cos bucket-create --bucket $BUCKET --ibm-service-instance-id $SERVICE_INSTANCE_ID --region $REGION
+----
+
+. Create `HMAC` credentials by running the following command. In the command, you pass `"HMAC":true` as a parameter.
++
+[source,terminal]
+----
+$ ibmcloud resource service-key-create test-key Writer --instance-name test-service-instance --parameters {\"HMAC\":true}
+----
```

Comment:
```suggestion
. Create the `HMAC` credentials by running the following command: 
+
[source,terminal]
----
$ ibmcloud resource service-key-create <service_key_name> Writer --instance-name <service_instance_name> --parameters {\"HMAC\":true}
----
```

- The following sentence can be removed: "In the command, you pass `"HMAC":true` as a parameter." Because the command uses the `--parameter` flag so IMO it's self-explanatory. But if you want to explain `"HMAC":true`, you can use a call out.
- The command may use placeholders instead of example names.

Reply:
Removed the parameter description.